hive-metastore/src/main/java/org/apache/iceberg/hive/HiveOperationsBase.java

```diff
@@ -169,8 +260,8 @@ default Table newHmsTable(String hmsTableOwner) {
             null,
             Collections.emptyList(),
             Maps.newHashMap(),
-            null,
-            null,
+            properties.getOrDefault(HiveCatalog.VIEW_ORIGINAL_TEXT, null),
```

Comment:
why is this adding view-specific things?

Reply:
I think this might not be specific to Hive-View.  This is another field in Hive-Table .  IMO, This field either we can ignore for both iceberg-table and view or make it optional with properties. 
Ref: https://github.com/apache/hive/blame/df45194268ffb10f9f7aae0ab4e3aec35a7b31d8/standalone-metastore/metastore-common/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/metastore/api/Table.java#L23

Reply:
I think this PR should be a pure refactoring and not add new things/functionality. The current code on `main` doesn't look at this property, so why do we need to check this property as part of this PR?